src/CommunityToolkit/Xamarin.CommunityToolkit/Views/TabView/TabView.shared.cs

```diff
@@ -958,11 +958,27 @@ void UpdateIsTabStripVisible(bool isTabStripVisible)
 
 		void UpdateTabContentHeight(double tabContentHeight) => contentContainer.HeightRequest = tabContentHeight;
 
-		void UpdateTabIndicatorColor(Color tabIndicatorColor) => tabStripIndicator.BackgroundColor = tabIndicatorColor;
```

Comment:
`tabStripIndicator` is guaranteed not to be null here since it's readonly and set in the constructor (line 59).
Same comment for other three `tabStripIndicator` checks bellow.

Reply:
As mentioned in the linked issue when you set the properties from App.xaml, you will hit that case. Not sure why but I think `OnTabIndicatorColorChanged()` may be called from [here](https://github.com/xamarin/XamarinCommunityToolkit/blob/2e700879040e68c7288fa8be0a4d91bfd0f4b0d3/src/CommunityToolkit/Xamarin.CommunityToolkit/Views/TabView/TabView.shared.cs#L61) at that point` tabStripIndicator` is not instantiated yet. The same analogy could be applied to your other comment.

Reply:
`OnTabIndicatorColorChanged` is called, but `tabStripIndicator` is not null when it executes

![image](https://user-images.githubusercontent.com/6609929/111051602-f3f96e80-845c-11eb-9648-6778dd0a5551.png)

I tried to reproduce the issue (set the style in `App.xaml` and opened TabedPage in sample app), but couldn't. Are you sure this issue is still present?

Reply:
As we discussed on the chat, I tried to reproduce the issue on the current develop branch and confirmed that it is still happening, for anyone who want to test add below style to App.xaml and open the "TabView>Custom Tabs" page.
```
   <Style TargetType="{x:Type xct:TabView}">
            <Setter Property="TabIndicatorColor" Value="{StaticResource AppBackgroundColor}"/>
            <Setter Property="IsTabTransitionEnabled" Value="False"/>
            <Setter Property="TabIndicatorWidth" Value="55"/>
            <Setter Property="TabIndicatorHeight" Value="10"/>
</Style>
```